scripts/documentation.js

```diff
@@ -0,0 +1,33 @@
+const shellUtils = require('shell-utils');
+const exec = shellUtils.exec;
+const fs = require('fs');
+
+export function release(version, override = false) {
+  if (override) _removeDocsVersion(version);
```

Comment:
If version 7.0.0 was already published and we want to update it - do we need to pass `override = true`? Can we detect if 7.0.0 was already published (by reading versions.json) and deleting it before generating it again?